src/index.js

```diff
@@ -159,6 +158,11 @@ class Dropzone extends React.Component {
       }
     });
 
+    if (!multiple) {
+      // if not in multi mode add any extra accepted files to rejected.  This will allow end users to easily ignore a multi file drop in "single" mode.
+      rejectedFiles.push(...acceptedFiles.splice(1));
```

Comment:
Change acceptedFiles to a `let` so it can be redefined and then do something like 

```
const [accepted, ...rejected] = acceptedFiles;
rejectedFiles.push(...rejected);
acceptedFiles = [accepted];
```

Or just leave it as it is?